### PR TITLE
Add github documentation and fill in missing information

### PR DESCRIPTION
**Pull request by Christoffer-Cortes**
### Applicable Issues
Closes https://github.com/eiffel-community/eiffel-gerrit-plugin/issues/4

### Description of the Change
Fill in missing information in the plugin about.md and link to that documentation in github REAME.

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,3 +39,173 @@ scalability, flexibility and traceability. Eiffel is based on the concept of dec
 messaging, both to drive the continuous integration and delivery system and to document it.
 
 Visit [Eiffel Community](https://eiffel-community.github.io) to get started and get involved.
+
```
Comment (a-breid-ericsson) [inline README.md]: This file is just a duplication of the about.md could there not just be description and a link to about.md from here?

File: src/main/resources/Documentation/about.md [hunk 2]
```diff
@@ -98,12 +98,71 @@ or by editing project configuration using web ui.
...
 --------------------
 
-TODO: Explain event linking and show examples.
+When a patchset is pushed to a gerrit repository and branch the EiffelSourceChangeCreatedEvent that is created will have the
+BASE link set to the previous EiffelSourceChangeSubmittedEvent. If there is no previous EiffelSourceChangeSubmittedEvent 
+for that particular repository and branch then no BASE link will be set. Any following patchsets pushed to the same review will
+be connected in a one way chain from newest to oldest through the PREVIOUS_VERSION link.
+When it's time to submit the patchset an EiffelSourceChangeSubmittedEvent is created with a CHANGE link set to the last
+patchset pushed to that particular review and a PREVIOUS_VERSION link set to the last EiffelSourceChangeSubmittedEvent
+for that repository and branch. If there is no previous EiffelSourceChangeSubmittedEvent then no PREVIOUS_VERSION link will be set.
```
Comment (m-linner-ericsson) [inline src/main/resources/Documentation/about.md]: Is it easier to link to the scenario documentation and maybe move it?

File: README.md (hunk 1 above)
Comment (Christoffer-Cortes) [reply README.md]: Yes, I can do that, the formatting was rendered worse in github but I used the github formatting in the about.md instead and it looks a bit better now.

File: src/main/resources/Documentation/about.md (hunk 2 above)
Comment (Christoffer-Cortes) [reply src/main/resources/Documentation/about.md]: Sure, I will link to the scenarios doc.

**Pull request merged**